apps/dsp-app/cypress/support/commands/data-model-class-command.ts

```diff
@@ -0,0 +1,29 @@
+import { faker } from '@faker-js/faker';
```

Comment:
why do we have this command ? Can't we do it with an api request ?

Reply:
I think request is not the same. e2e should simulate user behaviour from start to end.